spec/introduction.md

```diff
@@ -855,16 +929,18 @@ using System;
 
 class Test
 {
-    static void Main() {
+    static void Main() 
+    {
         Entity.SetNextSerialNo(1000);
-        Entity e1 = new Entity();
-        Entity e2 = new Entity();
+        var e1 = new Entity();
+        var e2 = new Entity();
```

Comment:
I'm of two minds on this - while **var** is idomatic, many teams forbid its use. Also, there's a significant fraction of new C# developers who associate it with *untyped* variables (similar to C#'s **dynamic**, Visual Basics **variant**, or practically anything in JavaScript). Since this doc is an introduction to the language, being explicit with the variable type may reduce confusion.

Reply:
Sounds like a good reason to demonstrate `var` and its type-safe behavior early on.  If the introduction has to cater to all of the different opinions regarding code style it's going to only end up incredibly long and incredibly confusing.

Reply:
@theunrepentantgeek, This change follows the [coding style of coreFX](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md).

Reply:
Fair call. It may then be worth explicitly pointing out the use of **var** in the paragraph below.